mappings/net/minecraft/realms/RealmsError.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_4345 net/minecraft/realms/RealmsError
```

Comment:
Are these not mapped by Rune's PR?

Reply:
Maybe. In all honesty I haven't checked.

Reply:
Time to check!

Reply:
It is, as are a few other classes. I have removed them from this PR.